docs/01-Chart-Configuration.md

```diff
@@ -257,6 +257,18 @@ afterBody | `Array[tooltipItem], data` | Text to render after the body section
 beforeFooter | `Array[tooltipItem], data` | Text to render before the footer section
 footer | `Array[tooltipItem], data` | Text to render as the footer
 afterFooter | `Array[tooltipItem], data` | Text to render after the footer section
+dataPoints | `Array[CustomTooltipDataPoint], data` | List of matching point informations.
+
+#### CustomTooltipDataPoint
```

Comment:
I meant the snippet here https://github.com/chartjs/Chart.js/blob/master/docs/01-Chart-Configuration.md#tooltip-item-interface

I was also thinking to merge with the same section to that 2nd column of the last line of the table becomes `Array[TooltipItemInterface]`. In other words, merging the `CustomTooltipDataPoint` and `TooltipItemInterface` sections since they describe the same thing.

Also, for the tooltip item do we need the `x` and `y` properties? the user can look them up if necessary.